Vending machine structure

tenachev24 opened the pull request:
- building vending machine structure
- building product rows
- building product codes
- building product position

Commit:
- building vending machine structure
- building product rows
- building product codes
- building product position

nikolayshippii commented on src/components/VendingMachine.vue:
```diff
@@ -0,0 +1,237 @@
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
+@import "@/assets/css/style.css";
```
Ако това са стилове който ще се използват само в този комнпонент не е по-добре е да са import-нати, а направо написани в него (освен ако не са огромни, 600+ реда).
Също така е добре да изпозваш sass във всичи проекти го използваме.

tenachev24 replied:
От стиловете ще има още код, поне още 200 реда.

nikolayshippii commented on src/components/VendingMachine.vue:
```diff
@@ -0,0 +1,237 @@
+                </div>
+              </div>
+            </div>
+          </div>
```
markup-a трябва да се преработи, използвай container>row>item*n, flex-wrap на row-a

Commit:
- changing Component's name
- removing style.css
- moving styles in component
- installing Sass-loader
- using Sass convention in style
- installing node-sass

Commit:
- markup

Commit:
- styles and products
